apps/admin/src/styles.css

```
@@ -0,0 +1,645 @@
+:root {
+  color: #1f2520;
+  font-family: "Aptos", "Segoe UI", sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Aptos" (font-family-name-quotes)

(font-family-name-quotes)